helm/charts/radius/templates/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "radius.fullname" . }}
+  labels:
+    app: {{ include "radius.name" . }}
+    chart: {{ include "radius.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ include "radius.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "radius.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+          - name: http
+            containerPort: 8080
+            protocol: TCP
+          - name: radius-auth
+            containerPort: 1812
+          - name: radius-ac
+            containerPort: 1813
+        envFrom:
+          - configMapRef:
+              name: {{ include "radius.fullname" . }}
+        volumeMounts:
+          - mountPath: /opt/gluu/radius/logs
+            name: radius-logs
+        livenessProbe:
+          exec:
+            command:
+            - cat
+            - /tmp/gluu-radius-health
+          initialDelaySeconds: 30
+          periodDelaySeconds: 30 
+          failureThreshold: 20
+        readinessProbe:
+          exec:
+            command:
+            - cat
+            - /etc/gluu-radius-health
+          initialDelaySeconds: 25
+          periodDelaySeconds: 25
+          failureThreshold: 20
+      {{- if eq .Values.global.isDomainRegistered "false"}}
```

Comment:
missing something above related to this line

Reply:
Fixed